Update redirect and accessibility guidance (#4519)
DFE-Digital/get-into-teaching-app: docs/content.md

### Accessibility

#### Link text

All links should be descriptive and make sense in isolation.
 

Avoid:
* including punctuation in link text 
* anchor links
* using the same link text to link to different places
* linking to the same place more than once, particularly if using different link text
* having two or more hyperlinks close together on the page

By default, all links will open in the same tab. You should only open a link in a new tab if the user will risk losing key information on the page. For example, a link to more information in a form. 

Links that direct to external sites (e.g. non-GOV.UK) should explicitly say their destination. 

[Find more guidance about writing accessible link text on GOV.UK](https://www.gov.uk/guidance/content-design/links).

#### Headings

The correct H1, H2, H3, H4 heading tags must be used to structure content. They should also be used in sequential order. For example, you must not skip from a H1 to using a H3. 

Heading text should also be unique and easy to distinguish from other headings on the page.

#### Contractions

Contractions can make content easier to read, but should be used consistently. 

> Postive contractions like you'll, it's and we've can be used on Get Into Teaching. 

Reviewer: ```suggestion
Positive contractions like you'll, it's and we've can be used on Get Into Teaching. 
```

### Accessibility

#### Link text

> All links should be descriptive and make sense in isolation

Author: ```suggestion
All links should be descriptive and make sense in isolation.
```

### Accessibility

#### Link text

All links should be descriptive and make sense in isolation
 

Avoid:
* including punctuation in link text 
* anchor links
* using the same link text to link to different places

> * linking to the same place more than one

Reviewer: ```suggestion
* linking to the same place more than once
```

* having two or more hyperlinks close together on the page

> * descriptive

Reviewer: does this need changing?

> * descriptive

Author: ```suggestion
```

By default, all links will open in the same tab. You should only open a link in a new tab if the user will risk losing key information on the page. For example, a link to more information in a form. 

Links that direct to external sites (e.g. non-GOV.UK) should explicitly say their destination. 

> [Find more guidance about writing accessible link text on GOV.UK]/(https://www.gov.uk/guidance/content-design/links).

Author: ```suggestion
[Find more guidance about writing accessible link text on GOV.UK](https://www.gov.uk/guidance/content-design/links).
```

Select the correct place from the drop down.

## Redirect URLs

A redirect is a way to send users to a different page to the URL they requested. We set up redirects where we do things like change the URL of a page, or remove a page. **If you need to set up a redirect from an external URL, these are handled in the routes file; you'll need developer support to create or amend these.**

To set up a redirect for a Get Into Teaching page, go to the 'redirects.yml' file. This file contains a list of all existing redirects. 

To make add a new redirect:

> 1. Check if the redirect you want to add fits into an existing redirect category. If not, create a new one.

Reviewer: Is this slightly confusing to talk about categories? Would it be better to say something like 'Redirects are grouped under headings based on which content they relate to. Check if the redirects fits into an existing heading/section (?). If not, create a new one.

> 1. Check if the redirect you want to add fits into an existing redirect category. If not, create a new one.

Author: ```suggestion
1. Redirects are grouped under headings, based on the content they relate to. Check if the redirect you want to add fits under an existing heading. If not, create a new one.
```
